### ISSUE TITLE
Post Data

### ISSUE THREAD

**Issue by jacobgqc** [glos/IoH2O#2]
We have historical and live data (and the necessary metadata) that we would like to post using JSON rest.
Will we need some credentials, and probably documentation on the posting process?
Thanks.

**Comment by tslawecki** [comment]
Yes, you will need credentials and documentation. I'll bring in some colleagues who will be back next week for details, but I know one thing they'll request --- a basic description of your data, e.g.

"We have six sensors with hourly measurements of pH and temperature from 3/17/2017 to present"

**Comment by kkoch** [comment]
@jacobgqc  Here is a link to our metadata form http://www.glos.us/wp-content/uploads/2016/07/GLOS_Metadata_Form.pdf  which we will need filled out. Alternately if you already have ISO 19119/19139 (or even FDGC) metadata in XML format and can provide a link I can see if our repository will import.  [Sometimes it works and sometimes we have to go back to the form depending on the XML formatting.]
